hash/ibm5170_cdrom.xml

```diff
@@ -7686,4 +7686,26 @@ Wave Studio v3.12
 		</part>
 	</software>
 
+	<software name="zorknemesis">
+		<description>Zork Nemesis: The Forbidden Lands</description>
+		<year>1996</year>
+		<publisher>Activision</publisher>
+		<info name="language" value="English" />
+		<part name="cdrom1" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="zork nemesis disc 1" sha1="85a6cac392f3648ba02320455690df1196b0bd79" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="zork nemesis disc 2" sha1="566903d9419cb929ed6fd4f230b71bbfb608245a" />
+			</diskarea>
+		</part>
+		<part name="cdrom3" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="zork nemesis disc 3" sha1="356fcaef31915484cb444529159cb9b69caefef5" />
+			</diskarea>
+		</part>
+	</software>
+
```

Comment:
* Can you try to make the `name` a bit more terse?  We try to jam them into eight characters if possible so there’s plenty of room for suffixes for clones if more versions show up.
* Please add `part_id` for each part to give human-readable disc labels.

Like:
```xml
		<part name="cdrom1" interface="cdrom">
			<diskarea name="cdrom">
				<feature name="part_id" value="What it says on the disc label" />
				<disk name="zork nemesis disc 1" sha1="85a6cac392f3648ba02320455690df1196b0bd79" />
			</diskarea>
		</part>
```

Reply:
Labels will be displayed when selecting the entry in File Manager. Pretty useful if the game specifically prompts for those.